addons/website_theme_install/static/src/less/website_theme_install.less

```diff
@@ -1,42 +1,162 @@
+@o-theme-kanban-gray: #fcfcfc;
+
 .o_kanban_view.o_theme_kanban {
+    background-color: @o-theme-kanban-gray;
+
+    /// Un-grouped Layout (default)
     &.o_kanban_ungrouped {
-        .o-justify-content(center);
+        .o-justify-content(space-between);
+
+        @media (min-width: @screen-md-min ) {
+            margin: 0 -@grid-gutter-width;
+        }
 
         .o_kanban_record {
-            .o-flex(0, 0, auto);
-            width: 100%;
-            @media (min-width: @screen-sm-min) {
-                width: 45%;
-            }
+            .o-flex(1, 0, 220px);
         }
     }
 
     .o_kanban_record {
+        margin-left: @grid-gutter-width;
+        margin-right: @grid-gutter-width;
         padding: 0;
+        box-shadow: none;
+        background: none;
+        border: none;
+
+        .o_browser_bar:before {
+            content: "•••";
+            display: block;
+            padding-left: 10px;
+            color: darken(@o-theme-kanban-gray, 10%);
+            font: 12px/20px Courier, monospace;
+            letter-spacing: -1px;
+        }
+
+        &.o_theme_installed .o_theme_preview_top {
+            border-color: @odoo-brand-optional;
+
+            > .fa-check-circle {
+                .o-position-absolute(10px, 10px);
+                .square(20px);
+                font-size: 20px;
+            }
+        }
+
+        .o_theme_preview_top {
+            border: 1px solid darken(@o-theme-kanban-gray, 20%);
+            border-radius: 2px;
+            .o-transform-origin(center, bottom);
+            .o-transition(all, .1s);
+
+            .o_theme_cover, .o_theme_logo, .o_theme_screenshot {
+                width: 100%;
+                padding-bottom: 127%;
+                background-repeat: no-repeat;
+                background-position: center center;
+                background-size: 40% 32%;
+            }
+
+            .o_theme_cover {
+                background-size: contain;
+            }
+
+            .o_theme_screenshot {
+                background-size: cover;
+            }
+        }
+
+        .o_theme_preview_bottom {
+            h5, h6 {
+                line-height: 16px;
+            }
+        }
 
-        position: relative;
-        overflow: hidden;
         .o_button_area {
-            .o-position-absolute(@bottom: -100px, @right: 0);
-            .o-transition(bottom, 400ms, ease);
-            width: 100%;
-            padding: @odoo-horizontal-padding/2;
-            background-color: fade(@odoo-brand-secondary, 70%);
-            text-align: right;
+            .o-position-absolute(20px, -1px, -1px, -1px);
+            .o-transition(opacity, 100ms);
+            .o-flex-display();
+            .o-flex-flow(column, wrap);
```

Comment:
It should definitely be `nowrap`... ^^
![theme_nowrap](https://user-images.githubusercontent.com/10338094/27737379-3b11aff6-5da8-11e7-9a80-3cccdb758e56.png)